src/components/web_progress_bar/web_progress_bar.ts

```diff
@@ -16,7 +16,7 @@ export class WebProgressBar extends BaseElement {
   render() {
     return html`
       <div class="web-progress-bar-wrapper">
-        <div class="web-progress-bar-indeterminate"></div>
+        <div class="web-progress-bar-indeterminate">isAwesome=${this.isAwesome}</div>
```

Comment:
If you want to delete this and the underlying property I think that's ok. I put it in there initially as an example of how to use the decorators.

Reply:
I mostly did this to test that it was working. Reverted now.